src/azul/openapi/schema.py

```diff
@@ -330,13 +330,19 @@ def in_range(minimum: Optional[N], maximum: Optional[N], type_: Optional[TYPE] =
 
 
 _primitive_types: Mapping[Optional[type], JSON] = {
-    str: {'type': 'string'},
     bool: {'type': 'boolean'},
-    int: {'type': 'integer', 'format': 'int64'},
+    dict: {'type': 'object'},
     float: {'type': 'number', 'format': 'double'},
+    int: {'type': 'integer', 'format': 'int64'},
+    list: {'type': 'array'},
+    str: {'type': 'string'},
     None: {'type': 'null'}
 }
 
+_primitive_strings: Mapping[str, JSON] = {
```

Comment:
bad name

Reply:
moved this reverse mapping into function where its used